docs/xostor/xostor.md

```diff
@@ -323,6 +323,120 @@ Only XCP-ng 8.2.1 is currently supported and with a maximum of 7 machines per po
 
 See this documentation: [RPU](/management/updates/#rolling-pool-update-rpu).
 
+## Upgrade
+
+If you are reading this documentation, we assume that you want to upgrade a pool on which XOSTOR is deployed, i.e. change the version of XCP-ng, for example from 8.2 to 8.3.
+For updates that don't change the version numbers (bugfixes, security fixes), see [the update section](#update).
+
+### 1. Prerequisites
+
+- All hosts must be up to date on the version of XCP-ng you are currently using. For this refer to [the update section](#update).
+- HA must be disabled on your pool.
```

Comment:
What kind of sanity checks on the LINSTOR cluster should be done before starting the upgrade process?

Reply:
I just added a new line about that.